pkg/apis/lighthouse/v1alpha1/types.go

```diff
@@ -47,6 +47,8 @@ const (
 	JobNameEnv = "JOB_NAME"
 	// JobTypeEnv is the type of job
 	JobTypeEnv = "JOB_TYPE"
+	// ContextNameEnv is the job's context
+	ContextNameEnv = "CONTEXT_NAME"
```

Comment:
Do we really need to change the CRD ?

Reply:
It’s not actually changing the CRD, just adding the context key...which I guess doesn’t need to be here. I originally was going to add it to the `GetEnvVars` function but changed my mind, so yeah, I’ll move this.